boards/shields/pmod_sd/boards/ek_ra8m1.overlay

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
```

Comment:
Same question- can this pin control be defined at the board level?

Reply:
I moved it into pin-ctrl at the board level.